src/minifluxapi.cpp

```diff
@@ -43,12 +45,21 @@ bool MinifluxApi::authenticate()
 
 	CurlHandle handle;
 	long response_code = 0;
-	run_op("/v1/me", json(), handle);
+	const std::string path = "/v1/me";
+	run_op(path, json(), handle);
 	curl_easy_getinfo(handle.ptr(), CURLINFO_RESPONSE_CODE, &response_code);
 
 	if (response_code == 401) {
 		return false;
 	}
+	if (response_code < 200 || response_code > 299) {
+		const auto url = server + path;
+		std::cerr << strprintf::fmt(
+				_("Authentication check using %s failed (HTTP response code: %ld)"),
+				url,
```

Comment:
I like to include this URL because it makes for an easy way to debug a few issues using regular browser or curl.

Current output:
![image](https://github.com/user-attachments/assets/cbed503d-2463-42ca-8977-335dd21019ee)

Wrong username/password output is not changed:
![image](https://github.com/user-attachments/assets/e3c93167-28f1-49e3-aa37-5a6c27947cec)